staging/src/k8s.io/apiserver/pkg/admission/plugin/validatingadmissionpolicy/validator.go

```diff
@@ -60,256 +48,28 @@ func (m *matchCriteria) GetMatchResources() v1alpha1.MatchResources {
 	return *m.constraints
 }
 
-// CELValidatorCompiler implement the interface ValidatorCompiler.
-type CELValidatorCompiler struct {
+type CELValidatingAdmissionPolicyValidator struct {
```

Comment:
Same as above. Consider omitting the term `ValidatingAdmissionPolicy` for types in a package of the same name.

Reply:
Maybe just `validator` + `NewValidator()` Validator`?

Reply:
Yes that's much better! Done.